index.html

```diff
@@ -4,8 +4,109 @@
     <meta charset="UTF-8" />
     <title>Fork Me! FCC: Test Suite Template</title>
     <link rel="stylesheet" href="css/style.css" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
   </head>
 
-  <body></body>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header" class="header">
+      <img
+        id="header-img"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+        alt="logo"
+      />
+      
+      <nav id="nav-bar">
+        <a href="#features" class="nav-link">Features</a>
+        <a href="#how-it-works" class="nav-link">How It Works</a>
+        <a href="#pricing" class="nav-link">Pricing</a>
+      </nav>
+    </header>
+
+    <section id="heroSection">
+      <h2 class="heroSection_title">Handcrafted, home-made masterpieces</h2>
```

Comment:
This should be in the h1 element
Let's use proper capitalization for the heading in CSS

![image](https://user-images.githubusercontent.com/43151556/155008814-fc66dccd-592b-46b9-9fea-71675fb6e4f7.png)